tests/unit-tests/system-status.php

```diff
@@ -93,7 +93,7 @@ public function test_get_system_status_info_database() {
 		$this->assertEquals( get_option( 'woocommerce_db_version' ), $database['wc_database_version'] );
 		$this->assertEquals( $wpdb->prefix, $database['database_prefix'] );
 		$this->assertEquals( WC_Geolocation::get_local_database_path(), $database['maxmind_geoip_database'] );
-		$this->assertArrayHasKey( 'woocommerce_payment_tokens', $database['database_tables'] );
+		$this->assertArrayHasKey( 'wptests_woocommerce_payment_tokens', $database['database_tables']['woocommerce'] );
```

Comment:
My test setup is a little different, with a different prefix, so this still fails. Maybe try `$wpdb->prefix . 'woocommerce_payment_tokens'` instead of "hard-coding" the prefix?

Reply:
Thanks for catching that!